doc/api/process.md

```diff
@@ -1819,6 +1819,46 @@ process.report.writeReport();
 
 Additional documentation is available in the [report documentation][].
 
+## process.resourcesUsage()
+<!-- YAML
+added: v12.x.x
+-->
+
+* Returns: {Object}
+    * `minflt` {integer}
+    * `majflt` {integer}
+    * `inblock` {integer}
+    * `oublock` {integer}
+    * `nvcsw` {integer}
+    * `nivcsw` {integer}
```

Comment:
I’m wondering if we should spell out these names fully in camelCase, e.g. `involuntaryContextSwitches`? I think the other diagnostic APIs on `process` might set enough precedent for that?

Reply:
I actually started with full names but i removed them later on, mainly because i believe it's easier for user to check what they each one actually means by checking against the kernel documentation. Otherwise we should document the mapping between both linux names and ours, what do you think ?

Reply:
I think using full names and documenting the mapping makes sense, but I’d also like to hear what other people think about it too.

Reply:
Seeing as this is mostly a *nix thing (only a couple or so properties are supported on Windows), I think it makes more sense to keep the original names used by the kernel.

Reply:
I'm a bit torn on this but I think I side with @addaleax because their names are cryptic even by unix standards.

Reply:
So what should we do ? I believe more people are in favor of using full names.

Reply:
:+1: for using full names.